src/main/java/com/prolinkli/core/app/components/user/service/UserAuthService.java

```
@@ -44,13 +45,14 @@ public AuthorizedUser login(UserAuthenticationForm userAuthForm) {
     }
 
```

The `login` method has several issues:

1. **Returns null on auth failure** - should throw `AuthenticationFailedException` instead of returning null
2. **Generic exception handling** -  Maybe? `RuntimeException` is too broad, should use a specific auth exception
3. **Missing user validation** - no check if `authForm.getUserFromCredentials()` returns null before JWT creation
4. **Inconsistent error handling** - some paths throw exceptions, others return null

Suggested improvements:
- Replace `return null` with proper exception
- Add null check for user after authentication
- Use specific authentication exceptions instead of generic RuntimeException (maybe)
- Consider logging authentication failures for security monitoring

---

fixed